apps/system/fxa/test/marionette/lib/fxa.js

```diff
@@ -149,9 +151,19 @@ FxA.prototype = {
     },
     clickFxaLogin: function() {
       var element = this.client.findElement(FxA.Selectors.fxaLogin);
+      console.log('found element', element);
       this.client.helper
           .waitForElement(element)
           .tap();
+      console.log('tapped');
+    },
+    clickFxsyncLogin: function() {
+      var element = this.client.findElement(FxA.Selectors.fxsyncLogin);
+      console.log('found element', element);
+      this.client.helper
+          .waitForElement(element)
+          .tap();
+      console.log('tapped');
```

Comment:
This does not have the effect of opening the FxA iFrame. When I do `document.querySelector('#get-started').click();` from the console, it does open the iFrame, so I'm not sure what's happening here...